### DISCUSSION TITLE
Is cache: "force-cache" being applied for <CldImage/>? Or does the image get fetched every refresh?

### DISCUSSION THREAD

**Discussion by IshfaqAhmedProg** [cloudinary-community/next-cloudinary#400] [Q&A]
Now that Next JS 13/14 has server actions what is the option for caching and are the images being cached? Or does the image load everytime the user refreshes the page? If not does anyone know how to set up caching for the images to keep the cloudinary cost low so that the images are not loaded in at every refresh, while using <CldImage/>?

**Comment by colbyfayock** [comment]
hey @IshfaqAhmedProg im not totally sure on the context of Server Actions here, but generally speaking, Cloudinary returns cache control headers which browsers will see and respect, where the next time a visitor visits the page those images will be loaded from memory instead, unless the browser determines not to for whatever reason including the user selecting to disable cache

<img width="1680" alt="image" src="https://github.com/cloudinary-community/next-cloudinary/assets/1045274/fbb16437-ace0-4345-9708-f584478b8ce0">

this really has nothing to do with Next.js though. this is a pretty general practice that you'll see with any image or static asset provider

**Reply by IshfaqAhmedProg** [reply]
I ask this because i have a site with multiple product images on the home page and it looks like my credit usage on cloudinary goes up everytime i refresh (ctrl+f5) the page on dev server. Im worried that when too many users access the website the credit usage might become very high. Is this something i should be worried about?

**Reply by colbyfayock** [reply]
is this publicly available taht i can look at it?

if you open up your network tab as i did in the screenshot above, make sure Disable Cache is **unchecked**, and refresh the page twice (to make sure the images are first loaded), does it show that the image was loaded from memory cache or disk? if so it's not making an external request
